.gitignore

```diff
@@ -8,3 +8,29 @@ local.nix
 # Misc
 *.swp
 devices/*/kernel/config.*.old
+
+### env ###
+.env
+.envrc
+.direnv/
+
```

Comment:
It's a bit different than the next section, for those files. They are files that are realistically to be found in a repository.

Though, I don't think it's relevant to "guard" against files a user might add but are not desired by the project (at this point).

Looks like you could use [`.git/info/exclude`](https://stackoverflow.com/questions/22906851/when-would-you-use-git-info-exclude-instead-of-gitignore-to-exclude-files)

Reply:
That makes sense. I wasn't aware of .git/info/exclude so thank you for the information!

Reply:
To be fair, I didn't either before looking into it. I wondered if there was a way, and yes there is!

It'll be useful for my own use too.